bugbot/rules/workflow/no_severity_nag.py

```diff
@@ -73,6 +74,8 @@ def handle_bug(self, bug, data):
             # check if the product::component is in the list
             utils.check_product_component(self.components_skiplist, bug)
             or utils.get_last_no_bot_comment_date(bug) > self.activity_date
+            or any(keyword in SECURITY_KEYWORDS for keyword in bug["keywords"])
+            or bug["groups"]
```

Comment:
Maybe we could have this in the query directly

Reply:
How does this look? 55a1b1f684ab36ec3ec435f54d73877f85adb2fa

Reply:
You could instead add it as a configuration, and it will be added to the query:
https://github.com/mozilla/bugbot/blob/a8b33bedcf4c7cf0e0deb5448b2748912ffe9ea7/configs/rules.json#L452-L452

Reply:
If you add `"sec": false` config, the query will automatically exclude security bugs?

See:
https://github.com/mozilla/bugbot/blob/8ede0c2b3a2721c9aa6fe2766fa53146261ef47c/bugbot/bzcleaner.py#L229-L230

https://github.com/mozilla/bugbot/blob/8ede0c2b3a2721c9aa6fe2766fa53146261ef47c/bugbot/bzcleaner.py#L439-L441

Reply:
Done here: 31364910f6cb88aee0214a838bd9d59c68130c65